### ISSUE TITLE
Feature Idea: Include livereload code in error report pages  

### ISSUE THREAD

**Issue by netcarver** [baumrock/RockFrontend#25]
I don't know how easy this would be - but could the livereload JS be included in error report pages (inc tracy) when in debug mode?

This would allow a change to the code that fixes the error to cause the page to reload - if the change doesn't fix the error, or produces a different one, then the error page will be suitably regenerated anyway.

**Comment by BernhardBaumrock** [comment]
Hi @netcarver I've had that already working at some point but refactoring broke it :) I'll have another look!

**Comment by netcarver** [comment]
@BernhardBaumrock Thanks for getting this sorted - it's a real improvement. Closing this now.

**Comment by BernhardBaumrock** [comment]
Yeah thx for the push - it's been annoying me for a while, but motivation is higher if I know that also others need something :)
